Makefile

```diff
@@ -87,3 +87,11 @@ show-data-model:
 	# With --dev, you'll see the currently experimental parts, as well.
 	# Use --help to learn more. 
 	./flexmeasures/data/scripts/visualize_data_model.py --uml --dev
+
+clean_db:
+	sudo -i -u postgres dropdb -U postgres --if-exists ${db_name}
```

Comment:
I feel that we should also create a db backup by default (possibly with a flag to override that). Using:
```
flexmeasures db-ops dump
```

Otherwise, we should add a warning when referring to this command in the documentation.

Alternatively, I saw the `dropdb` command has a `-i` flag that tells it to "prompt before deleting anything". Perhaps the latter is the way to go. What do you think, @nhoening?

Reply:
Using the -i flag might replace my idea of asking in the beginning for confirmation.
However, we'd probably still run the other commands. Maybe asking in the beginning is cleaner because we then exit right away.